sig-cli/charter.md

```diff
@@ -0,0 +1,159 @@
+ [two-thirds super majority] vote amongst current Subproject Owners. This
+  *SHOULD* be supported by a majority of subproject contributors (through
+ [lazy-consensus] with fallback on voting).
+  * There *SHOULD* be 3 to 5 Subproject Owners.
+  * *MUST* be specifically enumerated in an OWNERS file under the **owners**
+   label.  [sigs.yaml] *MUST* point to this OWNERS file under the **subproject ->
+   owners** label.
+
+* Approver
+  * *MUST* contribute to SIG CLI by reviewing and approving proposed changes to
+   the SIG CLI code for a directory and its sub-directories.
+  * New Approvers *SHOULD* be chosen from current Reviewers by a majority vote of
+   Subproject Owners for directories under their control.
+  * *MUST* be enumerated in an OWNERS file within a directory under the
+   **approvers** label.
+
+* Reviewer
+  * *MUST* contribute to SIG CLI by reviewing proposed changes to the SIG CLI
+ code for a directory and its sub-directories.
+  * *SHOULD* have made significant technical contributions during the review of at
+   least six PR's in the last six months.
+  * *MAY* be voted as Reviewer by a majority of current Subproject Owners for directories
+   and subdirectories under their control.
+  * *MUST* be enumerated in an OWNERS file within a directory under the
+   **reviewers** label.
+
+* Test Health Maintainers
+  * *MUST* contribute to SIG CLI by maintaining test health.
+  * *MUST* have successfully completed at least one test on-call rotation within the
+   last six months as shown in on-call schedule of [Test Playbook].
+
+* Member
+  * *MUST* show sustained contributions to at least one subproject or to the SIG.
+  * *MUST* maintain health of at least one subproject or the health of the SIG.
+  * *MAY* build new functionality for subprojects.
+  * *MAY* participate in decision making for the subprojects they hold roles in.
+  * The Chair(s), Technical Lead(s), and Subproject Owner(s) are Members.
+  * Approvers are Members.
+  * Reviewers are Members.
+  * Test Health Maintainers are Members.
+  * A significant SIG contributor *MAY* be chosen as a Member using
+   [lazy-consensus] amongst Subproject Owners with fallback to a majority vote
+   of Members.
+
+## Subprojects
+
+* [kubectl](https://github.com/kubernetes/community/blob/master/sigs.yaml#L555)
+
+### Subproject Creation
+
+* Subprojects *MUST* be created by [KEP] proposal and accepted by [lazy-consensus]
+of Technical Leads with fallback on majority vote of Technical Leads.
+The result *SHOULD* be supported by the majority of SIG Members.
+  * Subproject creation KEP *MUST* enumerate the founding Subproject Owners.
+  * [sigs.yaml] *MUST* be updated to include subproject information and
+   Subproject Owners.
+
+## Organizational management
+
+* SIG CLI meetings
+  * *SHOULD* be facilitated by Chairs unless delegated to specific Members.
+  * *SHOULD* be held bi-weekly on [Zoom](https://zoom.us/my/sigcli) video 
+ conference every other Wednesday at 9am PST.
```

Comment:
Please add a link to https://www.timeanddate.com/ so that's easier for folks from other TZ to convert that.